src/App.tsx

```diff
@@ -1,51 +1,27 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
-
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { Navigation } from './components/Navigation/Navigation';
```

Comment:
```suggestion
import { Navigation } from './components/Navigation';
```
since you have `index` file there